src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -50,6 +51,7 @@ trait ClusterComponent extends LeoComponent {
 
   object clusterQuery extends TableQuery(new ClusterTable(_)) {
 
+    private lazy val dummyDate:String = "1970-01-01T00:00:01.000Z"
```

Comment:
It looks like this is always parsed into an `Instant` -- could the constant just be `final val dummyDate =Instant.ofEpochMilli(1)`?

I don't think you need the `lazy` either.